sql/mysql/convert_test.go

```diff
@@ -584,23 +585,23 @@ func TestColumnTypeSpec(t *testing.T) {
 	}
 }
 
-func TestOverride(t *testing.T) {
-	s := schemautil.ColSpec("int", "int")
-	s.Overrides = []*schemaspec.Override{
-		{
-			Dialect: "mysql",
-			Version: "8",
-			Resource: schemaspec.Resource{
-				Attrs: []*schemaspec.Attr{
-					schemautil.StrLitAttr("type", "bigint"),
-				},
-			},
-		},
-	}
-	d := &Driver{version: "8.11"}
-	c, err := d.ConvertColumn(s, nil)
-	it, ok := c.Type.Type.(*schema.IntegerType)
-	require.True(t, ok)
-	require.NoError(t, err)
-	require.Equal(t, "bigint", it.T)
-}
+//func TestOverride(t *testing.T) {
+//	s := schemautil.ColSpec("int", "int")
+//	s.Overrides = []*schemaspec.Override{
+//		{
+//			Dialect: "mysql",
+//			Version: "8",
+//			Resource: schemaspec.Resource{
+//				Attrs: []*schemaspec.Attr{
+//					schemautil.StrLitAttr("type", "bigint"),
+//				},
+//			},
+//		},
+//	}
+//	d := &Driver{version: "8.11"}
+//	c, err := d.Column(s, nil)
+//	it, ok := c.Type.Type.(*schema.IntegerType)
+//	require.True(t, ok)
+//	require.NoError(t, err)
+//	require.Equal(t, "bigint", it.T)
+//}
```

Comment:
It looks like this PR is not ready for review yet, right?

Reply:
We currently don't support override or I would have migrated that as well.